biothings/hub/dataplugin/templates/manifest.yaml.tpl

```diff
@@ -0,0 +1,35 @@
+---
+version: '0.3'
+__metadata__: # Optional.
+  license_url: https://example.com/  # Optional. Put your license url here
+  licence: ABCXYZ # Optional. Your license name
+  url: https://example.com/  # Your site url
```

Comment:
@remoteeng00 let's add an optional `description` field under `__metadata__`. It can be used to provide an optional description text about this plugin